data/formats-data.js

```diff
@@ -4884,7 +4891,7 @@ exports.BattleFormatsData = {
 		tier: "BL2",
 	},
 	ducklett: {
-		randomBattleMoves: ["scald", "airslash", "roost", "hurricane", "icebeam", "hiddenpowergrass", "bravebird", "defog", "liquidation", "aquajet"],
+		randomBattleMoves: ["scald", "roost", "icywind", "bravebird", "defog", "liquidation", "aquajet"],
```

Comment:
ice beam > icy wind

Reply:
i probably forgot to do this at some earlier point